components/atoms/text/text.js

```diff
@@ -11,23 +11,27 @@ import styled, { css } from 'styled-components'
 
 export const Text = ({
   align,
+  bold,
```

Comment:
Please remove `bold` now that you added `weight`

Reply:
@majid-amiri i think having "bold" as props is handy, also it is used in iui website and maitai,
having <Text bold /> i think looks better than <Text weight='semiBold' />

Reply:
@mohsenParvar It doesn't make sense to me to have both `bold` and `weight`. Please remove `bold` and change the usages on other products as well.